lib/src/masked_text_controller.dart

```diff
@@ -193,26 +193,19 @@ class MaskedTextController extends TextEditingController {
   /// Removes the [mask] from the [value]
   String _removeMask(String mask, String value) {
     final result = StringBuffer('');
-    var maskCharIndex = 0;
-    var valueCharIndex = 0;
+    var charIndex = 0;
 
-    while (maskCharIndex != mask.length && valueCharIndex != value.length) {
-      final maskChar = mask[maskCharIndex];
-      final valueChar = value[valueCharIndex];
+    while (charIndex != mask.length && charIndex != value.length) {
```

Comment:
My only suggestion is that it would be safer to use `charIndex < mask.length && charIndex < value.length` in case some weird framework behavior.

Reply:
Done